templates/includes/sidebar.html

```diff
@@ -1,6 +1,12 @@
 <ul class="sidebar-nav">
-  <li><a href="{% url "taxi:index" %}">Home page</a></li>
-  <li><a href="{% url "taxi:manufacturer-list" %}">Manufacturers</a></li>
-  <li><a href="{% url "taxi:car-list" %}">Cars</a></li>
-  <li><a href="{% url "taxi:driver-list" %}">Drivers</a></li>
+    {% if user.is_authenticated %}
```

Comment:
The use of `{% if user.is_authenticated %}` is correct for checking if a user is logged in. Ensure that the `user` context variable is passed to the template in the views rendering this template.